lib/impure/db_postgres.nim

```diff
@@ -234,6 +234,10 @@ proc len*(row: InstantRow): int32 {.inline.} =
   ## returns number of columns in the row
   pqNfields(row.res)
 
+proc getColumnName*(row: InstantRow, col: int32) : string {.inline.} =
```

Comment:
nah, you convert it to int32 internally, in the proc. You cannot just change the interface that Postgres exposes.